openmc/stats/multivariate.py

```diff
@@ -12,6 +12,10 @@
 from openmc.stats.univariate import Univariate, Uniform
 
 
+__all__ = ['UnitSphere', 'PolarAzimuthal', 'Isotropic', 'Monodirectional',
+           'Spatial', 'CartesianIndependent', 'Box', 'Point']
```

Comment:
`__all__` specifies what attributes are imported whenever you have a `from _ import *` statement, so in general the purpose is to expose only certain attributes. My intent was really to avoid polluting namespaces, but since this only goes like 1% of the way (since it's not being used elsewhere), I'll just hold off and remove this. We can debate the merits of `__all__` in a separate PR.